### DISCUSSION TITLE
DAI 2024-2025 - Practical Work 1 - Dylan Langumier and Raphaël Perret

### DISCUSSION THREAD

**Discussion by maellemelody** [heig-vd-dai-course/heig-vd-dai-course#325] [Show and tell]
Creating a CLI application allowing to apply filters onto BMP files (and eventually PNGs if we have enough time)
Current goals:
- transparency changer
- moving average
- grayscale filter
- sepia filter

**Comment by GeraudSilvestri** [comment]
Seems good to me, go for it

**Comment by maellemelody** [comment]
Here's the repository link https://github.com/Dylan-Langumier/DAI-practical-work-1

**Comment by rp2709** [comment]
[Tag de rendu du premier travail pratique](https://github.com/Dylan-Langumier/DAI-practical-work-1/releases/tag/Rendu)

**Comment by maellemelody** [comment]
We also [released ](https://github.com/Dylan-Langumier/DAI-practical-work-1/releases/tag/Rendu) a ZIP archive file only containing the JAR file as well the BMP file we used for testing.
